src/tools/haxelib/Main.hx

```diff
@@ -951,39 +953,35 @@ class Main {
 			print("Checking " + p);
 			try doUpdate(p, state)
 			catch (e:Dynamic)
-				if (e != GIT_UNAVAILABLE) neko.Lib.rethrow(e);
+				if (e != VCS_UNAVAILABLE) neko.Lib.rethrow(e);
 		}
 		if( state.updated )
 			print("Done");
 		else
 			print("All libraries are up-to-date");
 	}
-	
-	static var GIT_UNAVAILABLE = 'Git unavailable';
+
+	static var VCS_UNAVAILABLE = 'Git unavailable';
 	function doUpdate( p : String, state ) {
 		var rep = state.rep;
-		if( FileSystem.exists(rep + "/" + p + "/git") && FileSystem.isDirectory(rep + "/" + p + "/git") ) {
-			checkGit();
-			if (!gitExists())
-				throw GIT_UNAVAILABLE;
+
+		//TODO: get content from `.current` and use vcs only if there "dev".
+
+		var vcs:Vcs = Vcs.getVcsForDevLib(rep + "/" + p);
+		if(vcs != null)
+		{
+			if(!vcs.exists)
+				throw VCS_UNAVAILABLE;
```

Comment:
This const never displayed as error if [Main.hx#L954-L956](https://github.com/fzzr-/haxelib/blob/feature/hg/src/tools/haxelib/Main.hx#L954-L956).
Okay. Do you want me to throw the instance of Vcs? O_o
I think i have to move `VCS_UNAVAILABLE` into Vcs and throw this const from it.
Or will create ENum VcsError maybe.